deploy/sre-prometheus/extended-logging/101-parsed_fluentd_openshift-logging_collector.PrometheusRule.yaml

```diff
@@ -31,7 +31,7 @@ spec:
             message: Prometheus could not scrape fluentd {{ $labels.instance }} for more than 10m.
             summary: Fluentd cannot be scraped
           expr: |
-            absent(up{job="collector"} == 1)
+            absent(up{job="collector"} == 1) or absent(up{job="fluentd"} == 1)
```

Comment:
Would this cause the alert to fire on both new/old versions of the Cluster Logging Operator?

i.e.
* New version: `absent(up{job="fluentd"} == 1` would fire
* Old version: `absent(up{job="collector"} == 1` would fire

Reply:
Fixed :)